M/MUMPS/MUMPS_seq@5/build_tarballs.jl

```diff
@@ -90,7 +90,7 @@ products = [
 dependencies = [
     Dependency(PackageSpec(name="CompilerSupportLibraries_jll", uuid="e66e0078-7015-5450-92f7-15fbd957f2ae")),
     Dependency(PackageSpec(name="METIS_jll", uuid="d00139f3-1899-568f-a2f0-47f597d42d70")),
-    Dependency(PackageSpec(name="libblastrampoline_jll", uuid="8e850b90-86db-534c-a0d3-1478176c7d93"))
+    Dependency(PackageSpec(name="libblastrampoline_jll", uuid="8e850b90-86db-534c-a0d3-1478176c7d93"), compat="5.2.0")
```

Comment:
Are you aware this is not going to work on Windows until Julia v1.9, right?

Reply:
I forgot that OpenBLAS and LBT are fixed for each Julia version.
Let's just compile MUMPS_seq with OpenBLAS32 on Windows platforms for now.